cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -110,6 +110,7 @@
    */
   public static final class HBase {
     public static final String AUTH_KEY_UPDATE_INTERVAL = "hbase.auth.key.update.interval";
+    public static final String MANAGE_COPROCESSORS = "manage.coprocessors";
```

Comment:
Can we rename this property to be something less likely to be collided with? Say `cdap.manage.hbase.coprocessors` or something...